packages/microsoft-reactnative-sampleapps/windows/SampleAppCPP/App.cpp

```diff
@@ -19,6 +19,12 @@ App::App() noexcept {
   MainComponentName(L"SampleApp");
   JavaScriptMainModuleName(L"index.windows");
 
+#ifdef BUNDLE
+  JavaScriptBundleFile(L"index.windows");
+  InstanceSettings().UseWebDebugger(false);
+  InstanceSettings().UseLiveReload(false);
+#endif
```

Comment:
These are default settings. I also find that this unnecessary makes the user code more complex. Can we avoid adding this code?

Reply:
Thinking about this change: it seems that it is a quite complex approach to create a new build configuration every time when we want to customize run of RN app. A better approach is when the ReactApplication can read parameters from a command line or from some other config file. That way we can provide customization as a part of CLI and/or debug parameters in VS. It would give us more flexibility and be aligned with RN on other platforms.